src/components/TopNavigationBar/TopNavigationBar.tsx

```diff
@@ -108,41 +131,132 @@ export default function TopNavigationBar({
                   getProps={({ isCurrent }) => ({
                     className: isCurrent
                       ? 'inline-flex items-center px-1 pt-0.5 border-b-2 border-blue-500 dark:border-blue-700 text-base font-medium leading-6 text-gray-900 dark:text-dark-high-emphasis focus:outline-none focus:border-blue-700 dark:focus:border-blue-500 transition'
-                      : 'inline-flex items-center px-1 pt-0.5 border-b-2 border-transparent text-base font-medium leading-6 text-gray-500 hover:text-gray-700 hover:border-gray-300 dark:text-dark-high-emphasis focus:outline-none focus:text-gray-700 focus:border-gray-300 transition',
+                      : 'inline-flex items-center px-1 pt-0.5 border-b-2 border-transparent text-base font-medium leading-6 text-gray-500 hover:text-gray-700 hover:border-gray-300  focus:outline-none focus:text-gray-700 focus:border-gray-300 dark:text-dark-high-emphasis dark:hover:border-gray-500 dark:focus:border-gray-500 transition',
```

Comment:
i think the other ones are `hover:text-gray-900 focus:text-gray-900` but this one is `hover:text-gray-700 focus:text-gray-700`